.buildkite/commands/test-ios.sh

```diff
@@ -49,7 +49,8 @@ fi
 
 set +e
 echo "$SECTION"
-npm run "$TESTS_CMD"
+# npm run "$TESTS_CMD"
+npm run test:e2e:ios:local
```

Comment:
I know this was just a test but we would also need to avoid exporting the following variables: `TEST_ENV` and `CIRCLE_BRANCH` so the tests run locally.